docs/runtimes/application.ipynb

```diff
@@ -82,16 +82,306 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Serving function code, save to /your/path/add_ten_model.py
</pre><p>or you can use magic</p>

<pre class="ql-syntax" spellcheck="false">%%writefile src/add_ten_model.py

</pre><p>or see <a href="https://docs.mlrun.org/en/latest/runtimes/mlrun_code_annotations.html#named-annotations" target="_blank">https://docs.mlrun.org/en/latest/runtimes/mlrun_code_annotations.html#named-annotations</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Better link is ./mlrun_code_annotations.html#named-annotations